testing/set-env-vars.sh

```diff
@@ -11,6 +11,12 @@ KUBEBUILDER_ASSETS="$(${ENVTEST} use ${ENVTEST_K8S_VERSION} --bin-dir ${LOCALBIN
 # not existing cluster by default
 USE_EXISTING_CLUSTER=${USE_EXISTING_CLUSTER:="false"}
 
+# if you plan to debug or run on existing cluster increase the timeout (30 seconds should be ok)
+SINGLE_TEST_TIMEOUT=${SINGLE_TEST_TIMEOUT:="5s"}
+
+# if you plan to debug or run on existing cluster increase the timeout (180 seconds should be ok)
+SUITE_TIMEOUT=30s
```

Comment:
```suggestion
export SUITE_TIMEOUT=${30s:-SUITE_TIMEOUT}
```
consider having it as a default here but also possible to be easily overwritten from the outside, e.g.
```
SUITE_TIMEOUT=180s make test
```

Reply:
And this is exactly a case which we have make test command has already SUITE_TIMEOUT set to 30. Agree.